consul/test_consul.py

```diff
@@ -720,3 +722,25 @@ def simple_integration_test(self):
         self.assertServiceCheck('consul.up')
 
         self.coverage_report()
+
+    def test_acl_forbidden(self):
+        """
+        Testing Consul Integration
+        """
+
+        config = {
+            "instances": [{
+                'url': 'http://localhost:8500',
+                'catalog_checks': True,
+                'network_latency_checks': True,
+                'new_leader_checks': True,
+                'catalog_checks': True,
+                'self_leader_check': True,
+                'acl_token': 'wrong_token'
+            }]
+        }
+        try:
+            self.run_check(config)
+        except HTTPError as e:
+            if e.errno == 403:
```

Comment:
@zippolyte we should actually assert the `e.errno == 403` to confirm the test is working as expected, otherwise if there were no exception, this test would also pass although the `acl_token` was actually bad.